src/routes_staff.py

```diff
@@ -18,7 +21,20 @@ def staff_home(netid):
             courses = db.get_courses_for_staff(netid)
             version_code = 'unknown'
             try:
-                git_hash = check_output('git rev-parse --short=8 HEAD'.split(' ')).decode('utf-8')
+                '''
+                    Below is a fix, to get the git versionCode displayed on the front end
+                    The workaround is to get the absolute path to the current directory, then cd into that folder, then run your git command.
+                    We need to do this, because many times we will start the service from outside the current directory.
+
+                    Additionally, you may need to modify the "Environment" variable in the "on-demand.service" file and append the path to where git is located.
+                    For example: Environment="PATH=path_to_venv:/usr/bin/" and /usr/bin is where git is located.
```

Comment:
These two lines of the changes in environment var in "on-demnad.service" is actually not needed. If the PATH env var was used by this subprocess, then we don't need to use the `env` to specify the `PATH=/usr/bin/` again. Let's simplify these to something like "this code  will only work in a linux like environment. It requires executable 
`env` and `git` to be in  `/usr/bin/`.